docs/console/docs/03-02-ui-api-layer.md

```diff
@@ -5,13 +5,15 @@ type: Details
 
 The UI API Layer is a backend service which provides an API for all views of the Console UI. This service consumes the Kubernetes API and exposes a simplified GraphQL API to allow frontends to perform Kubernetes resource operations.
 
+> **NOTE:** Read [this](/docs/root/components/security#details-graphql) security document for more information about the Kyma GraphQL implementation.
```

Comment:
```suggestion
> **NOTE:** Read [this](/docs/components/security#details-graphql) security document for more information about the Kyma GraphQL implementation.
```
remove `root`. The document type for security is `components`. `root` is the type for docs in `kyma/docs/kyma`.